Containerfile-dev

```diff
@@ -0,0 +1,9 @@
+FROM quay.io/389ds/ci-images:fedora
```

Comment:
Will do.   Also, I was wondering if the container build should be included in the default run instructions.  For the most flexibility I suppose it should.  For faster start-up it may be best to make it optional and default to using a pre-built image on DockerHub or quay.io (389ds account?).   Also to be considered is how to run the container build.  I generally have used GitHub action on [publish new release](https://github.com/JeffersonLab/dirsrv/blob/main/.github/workflows/docker-publish.yml) trigger.   I have generally make the default compose behavior to use an image and provide an optional build (via compose file precedence) which then requires [merging](https://docs.docker.com/compose/multiple-compose-files/merge/) two of three files: compose.yaml, compose.override.yaml, and build.yaml ([example](https://github.com/JeffersonLab/dirsrv/blob/main/docker-compose.override.yml), long winded [explanation](https://gist.github.com/slominskir/a7da801e8259f5974c978f9c3091d52c#how-does-this-work)).  Not clear how you want this handled.

Reply:
I've [added](https://github.com/389ds/ds-container/tree/main/devcontainer) `quay.io/389ds/devcontainer` image based on rawhide, it has 2 arches (amd64 and arm64) and is rebuilt nightly. This is enough to open a workspace in a devcontainer in VSCode. `devcontainer.json` covers some aspects of the compose (ports, mounts, etc), so not sure if it's really needed for this purpose.